build.gradle

```diff
@@ -46,7 +46,7 @@ dependencies {
     compile('com.google.inject:guice:4.1.0') {
         exclude(module: 'guava')
     }
-    compile('com.github.bisq-network.libdohj:libdohj-core:01ae7770') {
+    compile('network.bisq.libdohj:libdohj-core:01ae7770') {
```

Comment:
It requires changes in libdohj as well, no?

Reply:
Did you mean to ask whether changing the `groupid` like we see above requires changes in the libdohj build?

If so, the answer is no. The `network.bisq.*` groupids we're using here are a feature of JitPack we're taking advantage of. See https://jitpack.io/docs/#custom-domain-name, and see this DNS TXT entry we have set up in line with those docs:

```
$ dig txt git.bisq.network | grep github
git.bisq.network.       259     IN      TXT     "https://github.com/bisq-network"
```